src/index.html

```diff
@@ -7,6 +7,254 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="body__header header">
+      <img src="./images/logo.png" alt="logo">
+      <nav>
+        <ul class="header__nav nav">
+          <li class = "nav__item ">
+            <a class = "nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class = "nav__item">
+            <a class = "nav__link" href="#">Samsung</a>
```

Comment:
Перенось на новий рядок, тоді простіше буде читатися
```suggestion
            <a class = "nav__link" href="#">
              Samsung
            </a>
```